sensor_msgs/CompressedImage raw
---
sensor_msgs/CompressedImage result
